packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStylesComfyRefresh.ts

```diff
@@ -0,0 +1,232 @@
+          borderTopLeftRadius: 'inherit',
+          height: '100%',
+          left: '0',
+          top: '0',
+          width: pxToRem(3),
+        },
+      }),
+
+      ...(isNil(v.showActionMenu) &&
+        p.hasActionMenu && {
+          ':hover': {
+            [`& > .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
+          },
+          ...(p.showActionMenu && {
+            [`& .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
+          }),
+        }),
+
+      [`&:hover + .${chatMessageSlotClassNames.bubbleInset} .${chatMessageSlotClassNames.timestamp}`]: {
+        opacity: 1,
+      },
+
+      ...(p.attached === true &&
+        !v.isImportant && {
+          [p.mine ? 'borderTopRightRadius' : 'borderTopLeftRadius']: 0,
+          [p.mine ? 'borderBottomRightRadius' : 'borderBottomLeftRadius']: 0,
+        }),
+      ...(p.attached === 'top' &&
+        !v.isImportant && {
+          [p.mine ? 'borderBottomRightRadius' : 'borderBottomLeftRadius']: 0,
+        }),
+      ...(p.attached === 'bottom' &&
+        !v.isImportant && {
+          [p.mine ? 'borderTopRightRadius' : 'borderTopLeftRadius']: 0,
+        }),
+    };
+  },
+
+  bubbleInset: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    display: 'flex',
+    paddingTop: pxToRem(10),
+    paddingBottom: 0,
+    // use padding instead of margin so that the bubble container's :hover
+    // styles still apply when mousing over the gap between the container
+    // and bubble-inset.
+    paddingLeft: v.hasReducedHorizontalSpace ? pxToRem(2.5) : pxToRem(5),
+    paddingRight: v.hasReducedHorizontalSpace ? pxToRem(2.5) : pxToRem(5),
+    ...(p.mine ? { right: '100%', flexDirection: 'row-reverse' } : { left: '100%' }),
+
+    [`&:hover .${chatMessageSlotClassNames.timestamp}`]: {
+      opacity: 1,
+    },
+  }),
+
+  content: ({ props: p, variables: v, theme }): ICSSInJSStyle => ({
+    color: p.mine ? `var(--chat-bubble-fg-mine, ${v.contentColor})` : v.contentColor,
+    wordBreak: 'break-word',
+    wordWrap: 'break-word',
+    '& a': {
+      color: 'inherit',
+      textDecoration: 'underline',
+      wordBreak: 'break-all',
+      '&:hover': { textDecorationStyle: 'double' },
+      '&:focus': { textDecorationStyle: 'double' },
+    },
+    ...(p.failed && {
+      color: theme.siteVariables.colorScheme.default.foreground,
+    }),
+  }),
+
+  badge: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const styles: ICSSInJSStyle = {
+      position: 'relative',
+      top: pxToRem(-5),
+      width: pxToRem(25),
+      height: pxToRem(25),
+      backgroundColor: 'none',
+      color: v.isImportantColor,
+      zIndex: v.zIndex,
+      '& > :first-child': {
+        display: 'inline-flex',
+        margin: '0 auto',
+      },
+    };
+    if (p.mine) {
+      styles.marginRight = pxToRem(-5);
+    } else {
+      styles.marginLeft = pxToRem(-5);
+    }
+    return styles;
+  },
```

Comment:
```suggestion
    return {
      position: 'relative',
      top: pxToRem(-5),
      width: pxToRem(25),
      height: pxToRem(25),
      backgroundColor: 'none',
      color: v.isImportantColor,
      zIndex: v.zIndex,
      '& > :first-child': {
        display: 'inline-flex',
        margin: '0 auto',
      },
      
      marginLeft: pxToRem(-5),
      ...p.mine && ({
        marginLeft: 0,
        marginRight: pxToRem(-5),
      }),
    };
  },
```

Let's avoid mutations of styles